src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/TextDescriptor.php

```diff
@@ -339,6 +339,7 @@ protected function describeContainerDefinition(Definition $definition, array $op
      */
     protected function describeContainerAlias(Alias $alias, array $options = array())
     {
+        $options['output']->setDecorated(false);
```

Comment:
Thing is, this changes the state of the output for subsequent descriptions. Ie currently its expected to be always decorated. (https://github.com/chalasr/symfony/blob/bd8530572154cbab67a239e597501519d37fc588/src/Symfony/Bundle/FrameworkBundle/Tests/Fixtures/Descriptor/array_parameter.txt)